aws-iot-mitigationaction/src/test/java/com/amazonaws/iot/mitigationaction/CreateHandlerTest.java

```diff
@@ -0,0 +1,174 @@
+package com.amazonaws.iot.mitigationaction;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.jupiter.MockitoExtension;
+import software.amazon.awssdk.services.iot.model.CreateMitigationActionRequest;
+import software.amazon.awssdk.services.iot.model.CreateMitigationActionResponse;
+import software.amazon.awssdk.services.iot.model.ResourceAlreadyExistsException;
+import software.amazon.cloudformation.exceptions.CfnAlreadyExistsException;
+import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
+import software.amazon.cloudformation.proxy.Logger;
+import software.amazon.cloudformation.proxy.OperationStatus;
+import software.amazon.cloudformation.proxy.ProgressEvent;
+import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
+
+import static com.amazonaws.iot.mitigationaction.TestConstants.ACTION_ARN;
+import static com.amazonaws.iot.mitigationaction.TestConstants.ACTION_ID;
+import static com.amazonaws.iot.mitigationaction.TestConstants.ACTION_PARAMS_WITH_REPLACE_DEFAULT_POLICY_VERSION_PARAMS;
+import static com.amazonaws.iot.mitigationaction.TestConstants.CLIENT_REQUEST_TOKEN;
+import static com.amazonaws.iot.mitigationaction.TestConstants.DESIRED_TAGS;
+import static com.amazonaws.iot.mitigationaction.TestConstants.MITIGATION_ACTION_NAME;
+import static com.amazonaws.iot.mitigationaction.TestConstants.MITIGATION_ACTION_ROLE_ARN;
+import static com.amazonaws.iot.mitigationaction.TestConstants.MODEL_TAGS;
+import static com.amazonaws.iot.mitigationaction.TestConstants.SDK_ACTION_PARAMS_WITH_REPLACE_DEFAULT_POLICY;
+import static com.amazonaws.iot.mitigationaction.TestConstants.SDK_MODEL_TAG;
+import static com.amazonaws.iot.mitigationaction.TestConstants.SDK_SYSTEM_TAG;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class CreateHandlerTest {
```

Comment:
Let's add tests for every single action that we support in both CREATE and UPDATE paths

Reply:
Nevermind, saw that you added tests for translator